routers/admin/auths.go

```diff
@@ -81,6 +82,12 @@ func NewAuthSource(ctx *context.Context) {
 	ctx.Data["OAuth2Providers"] = models.OAuth2Providers
 	ctx.Data["OAuth2DefaultCustomURLMappings"] = models.OAuth2DefaultCustomURLMappings
 
+	ctx.Data["SSPIAutoCreateUsers"] = true
+	ctx.Data["SSPIAutoActivateUsers"] = true
+	ctx.Data["SSPIStripDomainNames"] = true
+	ctx.Data["SSPISeparatorReplacement"] = "_"
+	ctx.Data["SSPIDefaultLanguage"] = "en-US"
```

Comment:
Forcing a default language seems like the users will never get the default from their browsers. If any, the default should be Gitea's default language, which is not necessarily `en-US`.

Reply:
Agreed. Is it OK to use `ctx.Locale.Language()` instead?

It seems to be getting information from URL queries, cookies or 'Accept-Language`. But I am not sure if it defaults to Gitea's default language.

Reply:
May be `SSPIDefaultLanguage` should be empty by default. That way Gitea will determine the language from user's browser and `SSPIDefaultLanguage` will not interfere with this process.

And if the Gitea administrator knows his environment well and is sure that forcing a default language is OK, he select the correct default language for all users by changing `SSPIDefaultLanguage` to a non empty value.

The helper text for the field could contain an additional sentence like `Leave empty if you prefer the user's language to be automatically detected` to explain that.

This seems simple enough to me. Would that be OK?

Reply:
> 
> 
> May be `SSPIDefaultLanguage` should be empty by default. That way Gitea will determine the language from user's browser and `SSPIDefaultLanguage` will not interfere with this process.

That's perfectly acceptable. In fact, useful. 👍

Reply:
Implemented with commit https://github.com/go-gitea/gitea/pull/8463/commits/689872d6d1049b0eb00510f4e7572f4d19f2f165

`SSPIDefaultLanguage` is empty by default, which allows Gitea to determine the language when the user object is created.